.golangci.yml

```diff
@@ -59,3 +59,6 @@ issues:
   - linters:
     - golint
     text: "don't use ALL_CAPS in Go names; use CamelCase"
+  - path: test/e2e
+    linters:
+      - bodyclose
```

Comment:
I kept getting a bunch of false positives from this linter so I just turned it off for `test/e2e` but can revisit and explicitly ignore each occurrence if folks aren't comfortably with this. I don't think it's a big deal since it's just test code so any negative impact of not closing HTTP response bodies is limited to CI.